src/test/java/th/ac/kmitl/science/comsci/example/models/CityTest.java

```diff
@@ -8,6 +8,7 @@
     public void testInputCityName(){
         String cityName = "Bangkok";
         City ctyName = new City(cityName);
```

Comment:
instance object should be writing in Lower Camel Case while the Class should be writing in Camel Case.
```java
String cityName = "Bangkok";
City city = new City(cityName)
```